docs_user/modules/proc_adopting-image-service-with-object-storage-backend.adoc

```diff
@@ -25,8 +25,7 @@ spec
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably, MariaDB, {identity_service_first_ref} and {key_manager_first_ref}
-should be already adopted.
+* You have completed the previous adoption steps.
 
 .Procedure
 
```

Comment:
Step 1 “Write the patch manifest into a file” - Does this mean "create a new file called, for example, glance_swift.patch, and paste the following content into it"?
(Hackfest feedback) Step 1: The template is missing “secret:” Where does this go under “template:” and is the value “osp-secret”?

Reply:
Yes to both, and +1 to add the `osp-secret` template (it doesn't hurt to have it explicitly mentioned)